cmd/list.go

```diff
@@ -185,8 +187,14 @@ func showComponentVersions(env *meta.Environment, component string, opt listOpti
 	}
 	verList := []string{}
 	for v := range platforms {
+		if v0manifest.Version(v).IsNightly() {
+			continue
+		}
 		verList = append(verList, v)
 	}
+	if comp.Nightly != "" {
+		verList = append(verList, version.NightlyVersion)
```

Comment:
this may not show the nightly version that you had installed

Reply:
Yes, that's designed

Reply:
Because we store all vx.x.x-nightly+yymmdd to the directory "nightly", we don't know the exact version in this place